openmp/libomptarget/src/CMakeLists.txt

```diff
@@ -55,6 +55,22 @@ target_compile_definitions(omptarget PRIVATE
   DEBUG_PREFIX="omptarget"
 )
 
+target_compile_options(omptarget PRIVATE -fno-exceptions)
+
+if(NOT LLVM_ENABLE_RTTI)
+  target_compile_options(omptarget PRIVATE -fno-rtti)
+endif()
+
+include(CheckIPOSupported)
+check_ipo_supported(RESULT use_lto OUTPUT output)
+if(use_lto)
```

Comment:
Do we want LTO to be the default? This will always use it no matter what the user sets.

Reply:
That was my plan, yes. I don't see a reason not to. Compile time/memory is usually the only reason, and it should not apply here (too small).

Reply:
I suppose if we transition to this being a runtimes only build it will pretty much guarantee we always have LTO available. I'm fine with a default, but we really need a way to turn this off. LTO will make rebuilds for developers more annoying I feel.

Reply:
We check if it is supported, no?